### PR TITLE
Feature/legionairre nuc231 transfer page component

### PR DESCRIPTION
**Pull request by legionairre**
localhost_8082_components_page html

### CODE REVIEW

File: docs/components/page.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+# Page
+
+`<Page>` is a UI component that represents an application screen.
+
+---
+
+## Live Demo
+
+<DocExampleBox codeBox="https://codesandbox.io/s/n5y3lym66p?module=%2Fsrc%2FApp.vue">
```
Comment (ercang) [inline docs/components/page.md]: codebox url should be removed and you can add the live demo prop
< DocExampleBox :liveDemoMode="true" >

File: src/main.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ export { default as ActionBar } from './components/ActionBar';
 export { default as ActionItem } from './components/ActionItem';
 export { default as NavigationButton } from './components/NavigationButton';
 export { default as Label } from './components/Label';
+export { default as Page } from './components/Page';
```
Comment (ercang) [inline src/main.js]: this should be removed and Page should be imported where it is needed.

File: tests/unit/components/Page.spec.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { expect } from 'chai';
+import { mount } from '@vue/test-utils';
+import { Page, Label } from '../../../src/main';
```
Comment (ercang) [inline tests/unit/components/Page.spec.js]: page and label should be imported from the original path. it should be removed from main.js

File: src/main.js (hunk 2 above)
Comment (legionairre) [reply src/main.js]: Page has been removed from main.js

File: tests/unit/components/Page.spec.js (hunk 3 above)
Comment (legionairre) [reply tests/unit/components/Page.spec.js]: Page and label imported from the original path

File: docs/components/page.md (hunk 1 above)
Comment (legionairre) [reply docs/components/page.md]: line 13 has been changed

File: docs/.vuepress/components/PageDoc.vue [hunk 4]
```diff
@@ -0,0 +1,58 @@
+<template>
+  <Page :actionBarHidden="actionBarHidden">
+    <ActionBar title="My App" />
+    <StackLayout class="page-wrapper">
+      <StackLayout>
+        <Label text="My Content" />
+      </StackLayout>
+      <StackLayout orientation="horizontal">
+        <Label text="actionBarHidden:" />
+        <input type="checkbox" v-model="actionBarHidden" id="Page_actionBarHidden_input" />
```
Comment (mylmz10) [inline docs/.vuepress/components/PageDoc.vue]: Could you change input to TextField?

File: docs/.vuepress/components/PageDoc.vue [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+export default {
+  name: 'PageDoc',
+  data() {
+    return {
+      actionBarHidden: false,
+      backgroundSpanUnderStatusBar: false,
+      statusBarStyle: 'light',
+    };
+  },
+  methods: {
```
Comment (mylmz10) [inline docs/.vuepress/components/PageDoc.vue]: Navigate methods using only native. Please remove them? We should use own router plugin.

Comment (legionairre) [reply docs/.vuepress/components/PageDoc.vue]: They have been removed

File: tests/e2e/specs/web/components/searchbar.spec.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
```
Comment (mylmz10) [inline tests/e2e/specs/web/components/searchbar.spec.js]: Remove unnecessary file

Comment (legionairre) [reply tests/e2e/specs/web/components/searchbar.spec.js]: it has been removed

**Pull request merged**